### DISCUSSION TITLE
请教一个肿瘤化疗的问题

### DISCUSSION THREAD

**Discussion by zhaoliang0302** [ShixiangWang/self-study#36] [Q&A] [answered]
翔哥好，
我在读一篇“m6A RNA methylation regulators contribute to malignant progression and have clinical prognostic impact in gliomas”文章时有个小问题没有搞明白，特来请您指点。
![image](https://user-images.githubusercontent.com/42333702/101330648-d2ad4300-38ad-11eb-9a43-45b2ee8ab8d0.png)

图中展示了高风险患者相较于低风险患者可能对TMZ化疗更敏感，文中的高风险患者预后较差，分子病理特征、亚型也都是倾向于恶性表型，为什么高风险的患者对TMZ敏感但是预后却较差？同时，实际临床往往恶性程度较高的患者往往对化疗抵抗，化疗效果不理想。我不知道这种现象要怎么解释比较合理，我对于化疗了解还不够充分，找了文献也没有想要的答案，还请翔哥不吝赐，谢谢！

**Comment by ShixiangWang** [comment]
这个要两个图一起看，例如 F->G，G 图中都是高风险病人，但 with 或 without TMZ 区分度更明显，所以就是说对这个治疗更敏感。右侧的两个图是一个意思，不同的数据集而已。

**Comment by zhaoliang0302** [comment]
是的，我疑惑的是高风险组对化疗更敏感，为什么它的预后却更差？常识来说高风险组患者较恶性的分子特征应该对化疗抵抗才是，这里的结论不知道怎么解释

**Comment by ShixiangWang** [comment]
这里的比较不是你自己理解的那个意思。这里是把病人按风险分为高低两组，两组中针对 TMZ 的治疗效果是不一样的，高风险组里用药病人与不用药病人存在大的差异，而这个情况在低风险组里不存在。

**Comment by ShixiangWang** [comment]
另外这里没有比较过高风险组相比于低风险组的预后差异

**Comment by ShixiangWang** [comment]
就算有，高风险预后肯定要差的，不然还叫高风险组吗

**Comment by zhaoliang0302** [comment]
文中有说明高风险生存差于低风险，我的意思是高风险竟然对化疗敏感，低风险却不明显，这和常识有出入，一般高风险化疗抵抗。而且既然高风险化疗很敏感，为什么预后却比低风险差很多？2020年12月8日 09:52，Shixiang Wang <notifications@github.com>写道：

就算有，高风险预后肯定要差的，不然还叫高风险组吗

**Comment by ShixiangWang** [comment]
“高风险竟然对化疗敏感，低风险却不明显” 这个得看怎么定义这个风险值。它这里指的是**高风险组里 with TMZ 的组别对化疗更敏感**。

**Comment by ShixiangWang** [comment]
简化一下：你可以看作有 A 和 B 两组病人，A 组里面带 Z 标签的治疗效果比不带 Z 标签的好，但 B 组没有这种趋势，带不带 Z 标签治疗效果都差不多。没有其他额外的信息了。

**Comment by zhaoliang0302** [comment]
抱歉，图中的TMZ是化疗药，这个重要的信息我忘记说明了。

**Comment by ShixiangWang** [comment, marked as answer]
这个信息不影响理解这个图，重点是 H 比 F 的生存曲线分得开，我之前有篇文章图也是这样的

![image](https://user-images.githubusercontent.com/25057508/101446129-9cc79780-395d-11eb-8e1a-2562b82e82d5.png)

https://onlinelibrary.wiley.com/doi/full/10.1002/ijc.32327

**Comment by ShixiangWang** [comment]
太久我先关闭了。
